php/7.3-alpine/xdebug.ini

```diff
@@ -0,0 +1,11 @@
+xdebug.remote_enable=on
```

Comment:
The `xdebug.ini` for each version doesn't match, seems like this one has the more recent changes while the others don't. I assume we should be making them all consistent, can we just have a single `xdebug.ini`?

Reply:
@symric just a reminder on this before merging.